### DISCUSSION TITLE
Add option to make default primary save action "save as draft"

### DISCUSSION THREAD

**Discussion by groe** [craftcms/cms#8460] [Ideas]
(Follow-Up to #911)

### Description

For my client it would be useful to replace the "Save" button with "Save as a draft" button. Saving a draft is already possible via the save buttons dropdown menu, but they would like to prevent any accidental publishing of changes to **existing** pages as well, not only new ones.

So it would be great if there was a config setting to allow setting the default `saveEntryAction` to either `saveEntry` or `saveEntryDraft`, or something like that.

### Steps to reproduce

1. Edit an existing entry (or create a new one)
2. Click save button
3. Changes are now published, and not only saved as a draft

**Comment by FrisovanDijk** [comment]
I would like to add that the expected default behaviour when clicking 'Save' is _save and continue editing_. Right now it's _save and close_.

Being able to set the default behaviour is a good option to avoid such issues.
